iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/MergedDanglingLineXml.java

```diff
@@ -0,0 +1,90 @@
+package com.powsybl.iidm.xml;
```

Comment:
The copyright header and authorship are missing

Reply:
Fixed.